CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+> When contributing to this repository, please first discuss the change you wish to make via issue,
```

Comment:
> When contributing to this repository, please first discuss the change you wish to make with the owners of this repository before making any changes. You can contact the owners by email or via an issue.

or if it is truly about any means of communication then this is enough --

> When contributing to this repository, please first discuss the change you wish to make with the owners of this repository before making any changes.

Reply:
Latter sounds good. Will update.

Reply:
@B1ggDave  Updated via https://github.com/JetBrains/compose-jb/pull/511/commits/59f6636402682a1f8825ccfc0a74c2bc0b8c3813